CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 8.0.13
```

Comment:
I actually haven't released 8.0.12 yet, so you can put these notes under 8.0.12

Reply:
Done, thanks!